### ISSUE TITLE
Prometheus Plugin Failed. Error: Loading Chunk 44 failed. (missing...... prometheusPlugin....js)

### ISSUE THREAD

**Issue by mpryhu** [grafana/grafana#21733]
I have Grafana 6.5.3 installed. 
Every device Grafana is working (iOS, Android, Firefox) except Apple iPad 2 Wi-Fi + 3G with iOS 9.3.5 (resolution 768x1024). 
When loading Grafana Graph there is an Error: 
Prometheus Plugin Failed 
Error: Loading chunk 44 failed. (missing:http://[ip path]/public/build/prometheusPlugin.2ffa3667c2c8a7a8351f.js)

Why this error occurs? Is there any way to elimanate it?

**Comment by aknuds1** [comment]
Can you please use our issue template?

**Comment by mpryhu** [comment]
Can you please how to do it and explain where can I find information for using a template issue ? In Grafana or Girlthub? I am fresh and dont know it :( I am using Grafana on Raspberry Pi Raspbian :)

**Comment by marefr** [comment]
@mpryhu Could be related to your browser doesn't support certain things. Think this may have been fixed by #21684. Please try 6.6.0-beta1 or a nightly version. 6.6 stable should soon be released. Let us know if it still doesn't work and we'll consider reopen this issue.

**Comment by mpryhu** [comment]
Hello,
I just upgraded to 6.6.0 (official) and still having the same issue.... :( 
Do I have to install any other version?

**Comment by marefr** [comment]
@mpryhu do you get the same error browsing this dashboard on your device? https://play.grafana.org/d/000000029/prometheus-demo-dashboard?orgId=1&refresh=5m

**Comment by mpryhu** [comment]
![Zdjęcie 28 01 2020, 16 22 21](https://user-images.githubusercontent.com/60251149/73277706-d7638e00-41ea-11ea-86de-52c2e2512a22.png)
:( manny errors .....

**Comment by marefr** [comment]
Could be that your browser is too old. I may have an old Ipad 2 at home so can try to test it.

**Comment by ErickStaal** [comment]
Same error. However, I'm using the latest version of Firefox on Fedora 31. Hard to believe that this browser version would be too old. Btw in the local build directory on my monitoring system the following plugin version is present: prometheusPlugin.eee52fe17bb0c79cf513.js is present. However it doesn't seem to be called. Version of grafana is v6.6.0 from grafana rpm repo.

**Comment by condemime** [comment]
Hi, same error on Samsung Monitor DBE whith built-in MagicInfo Playser S3.
Tried with Testdata DB source : it's fine.
It fails with prometheus data source...

**Comment by dalemorson** [comment]
We get the same error with a Slack webhook message: 
![image](https://user-images.githubusercontent.com/46656789/74446818-61a92480-4e70-11ea-8e2e-d6a2267ed6c4.png)

Full error:

```Prometheus Plugin Failed
Error: Loading chunk 44 failed. (missing:http://localhost:3000/public/build/prometheusPlugin.2ffa3667c2c8a7a8351f.js)
```

**Comment by ErickStaal** [comment]
After updating to grafana v6.6.1 and rebooting the monitoring system the dashboard works again as advertised.

**Comment by dalemorson** [comment]
We can't upgrade to v6.6.1 at the moment due to the following https://github.com/grafana/grafana/issues/21868

**Comment by dalemorson** [comment]
There is a PR open that will help us move to 6.6.1: 

https://github.com/grafana/grafana/pull/22035

**Comment by marefr** [comment]
Was able to reproduce this using PhantomJS on v6.6.1. Will try to fix and include in v6.6.2

**Comment by marefr** [comment]
PhantomJS issue fixed by #21677. Will be included in upcoming v6.6.2 release.
